dev/releases/release_notes.readme.md

```diff
@@ -52,36 +67,10 @@ Currently the following labels are regarded as relevant for the creation of rele
   I think that these two labels are obsolete in the new workflow where one can automatically create the current state of the `CHANGES.md` section about the forthcoming release **at any time**.
 
 - The above labels describe whether a pull request is relevant at all, and if yes then how it is treated.
-  The following labels describe the ordering of the groups of pull requests and the subheadings of these groups.  Each pull request gets assigned to the **first** group/subheading that corresponds to one of its labels.
-
-| | subheading | labels | meaning |
-| - |------------- | ------- | -------- |
-| 1 | New features and major changes | `release notes: highlight` | PRs introducing changes that should be highlighted at the top of the release notes |
-| 2 | Fixed bugs that could lead to incorrect results | `kind: bug: wrong result` | PRs fixing bugs that result in mathematically or otherwise wrong results |
-| 3 | Removed or obsolete functionality | `topic: obsolete functionality` | PRs changing `lib/obsolete.g*` (move functions there or remove something from there) |
-| 4 | Fixed bugs that could lead to crashes or unexpected error messages | `kind: bug: crash` | PRs fixing bugs that may cause GAP to crash |
-| 4 | Fixed bugs that could lead to crashes or unexpected error messages | `kind: bug: unexpected error` | PRs fixing bugs that may cause GAP to run into an unexpected error |
-| 5 | Other fixed bugs | `kind: bug:` | PRs fixing bugs |
-| 6 | Improved and extended functionality | `kind: enhancement` OR `kind: new feature` OR `kind: performance` | PRs implementing enhancements |
-| 7 | Improvements in the experimental way to allow 3rd party code to link GAP as a library (libgap) | `topic: libgap` | PRs that are related to libgap |
-| 8 | Improvements in for the **Julia** integration | `topic: julia` | PRs that are related to the Julia integration |
-| 9 | Changed documentation | `topic: documentation` | PRs improving the documentation |
-| 10 | Packages | `topic: packages` | PRs related to package handling, or specific to a package (for packages w/o issue tracker) |
-| 11 | Other changes | `release notes: needed` OR `release notes: added` | PRs relevant for the release notes but without a label that fits --in each case, think about assigning such a label or introducing a new one |
+  The script `generate_release_notes.py` groups the pull request based on labels in a list called  `prioritylist`. Each pull request gets assigned to the **first** group/subheading that corresponds to one of its labels. This list can be adjusted over time as needed. Note that any such ordering is always a compromise, there is no best way to do it.
```

Comment:
Apparently it makes no sense to give an explicit list of the relevant labels, since this list changes.
Perhaps replace `This list can be adjusted over time as needed.` by an explicit sentence that before processing the relevant pull request with `generate_release_notes.py`, one should fetch the list of labels in question, and adjust the script accordingly.

Reply:
Thanks for the feedback. My apologies, but I don't understand the suggestion: from where should be one "fetch the list of labels in question" ? I am not sure what I am supposed to write here, a concrete suggestion would perhaps clarify this...

That said, the view I see it, both the set of labels, how we apply them, *and* the choice which of them are used for grouping, and how, are all in flux. So one may argue that instead of fetching labels from elsewhere, one might as well add/change/remove labels elsewhere... So what we really need is some clear concise guidelines how to apply labels, and which labels are used (to avoid the list of de-syncing, this could simply point at the relevant portions of the script)

Reply:
As I need to move forward with the release process, I am going to merge this PR now; that's not intended to shut down this discussion, though; we can still adjust this later on, after all

Reply:
@fingolfin Sorry for the confusion.
I wanted to say that creating release notes with `generate_release_notes.py` relies on the labels that are actually used in the relevant pull requests; some of them may be new, some labels used in the script may have been renamed, etc.
Thus the first step should be to fetch all labels from the pull requests. Perhaps `generate_release_notes.py` should be extended to collect these labels, compare this list with the labels which it knows already from its `prioritylist`, and then output the newly found labels. Before the next run of `generate_release_notes.py`, `prioritylist gets adjusted.